packages/smooth_app/lib/pages/scan/search_panel.dart

```diff
@@ -13,27 +13,25 @@ class SearchPanel extends StatefulWidget {
 }
 
 class SearchPanelState extends State<SearchPanel> {
+  final TextEditingController _searchFieldController = TextEditingController();
   final FocusNode _searchFieldFocusNode = FocusNode();
   final PanelController _controller = PanelController();
```

Comment:
I know its not from this PR but with multiple controllers maybe something like `_panelController` instead